packages/nativescript-feedback/index.android.ts

```diff
@@ -10,7 +11,8 @@ export class Feedback extends FeedbackCommon {
   show(options: FeedbackShowOptions): Promise<void> {
     return new Promise<void>((resolve) => {
       this.lastAlert = null;
-      const alerter = com.tapadoo.alerter.Alerter.create(Application.android.foregroundActivity)
+      const activityOrDialog = options.android?.dialog ? (options.android.dialog as View)._dialogFragment?.getDialog() : Application.android.foregroundActivity;
```

Comment:
Wouldn't it be possible to use `instanceof` here so the cast can be dropped?

For example:
```ts
const activityOrDialog = options.android?.dialog instanceof View ? options.android.dialog._dialogFragment?.getDialog() : Application.android.foregroundActivity;
```

Reply:
`_dialogFragment` is a public member of the `View` class exported by the android specific implementation, and not available in the index.d.ts. Casting as `<any>` would be the alternative, since the only purpose is to reference the android-specific definition without incurring vscode's wrath. In which case, the import can be removed entirely, but the intent and origin would be obfuscated.

Reply:
but, yes. I think you're correct that the cast could be dropped if `instanceof` is used.... i think i was hedging there.